sandbox/modules/identity/src/internalClusterTest/java/org/opensearch/identity/UserCrudIT.java

```diff
@@ -35,16 +37,32 @@ public void startClusterWithIdentityIndex() throws Exception {
     @SuppressWarnings("unchecked")
     public void testUsersRestApi() throws Exception {
 
+        final Map<String, String> emptyMap = Map.of();
+        final List<String> emptyList = List.of();
+
+        final RequestOptions authHeaderOptions = RequestOptions.DEFAULT.toBuilder()
+            .addHeader("Authorization", "Basic YWRtaW46YWRtaW4=")
+            .build(); // admin:admin
+
         String username = "test-create";
-        String requestContent = "{ \"password\" : \"test\","
+        String createContent = "{ \"password\" : \"test\","
             + " \"attributes\": { \"attribute1\": \"value1\"},"
             + " \"permissions\": [\"indices:admin:create\"]"
             + " }\n";
 
+        String updateContent = "{ \"password\" : \"test\","
+            + " \"attributes\": { \"attribute1\": \"value2\"},"
+            + " \"permissions\": [\"indices:admin:update\"]"
```

Comment:
Since permissions is set to be removed from the user object can this test other attributes of a user?

Reply:
We should address this as part of cleanup of permissions from user.